src/components/ItemDetail/DetailSummary/SummaryListItem.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import SummaryListItemData from './SummaryListItemData';
+
+const SummaryListItem = props => {
+  const { header, items } = props;
+  return (
+    <div className="summary-list-item">
+      <div className="data-label">{header}</div>
+      {items.map((item, index) => (
+        <SummaryListItemData key={index} item={item} />
```

Comment:
If you wanted, could bypass the `<SummaryListItemData />` component and just return `<div className="data-value">{item}</div>`.  Unless you're anticipating the `<SummaryListItemData />` to grow in complexity...